addons/escoria-core/game/core-scripts/esc/esc_event_manager.gd

```diff
@@ -5,11 +5,14 @@ class_name ESCEventManager
 # Emitted when the event started execution
 signal event_started(event_name)
 
+# Emitted when an event is started in a channel of the background queue
+signal background_event_started(channel_name, event_name)
+
 # Emitted when the event did finish running
-signal event_finished(event_name, return_code)
+signal event_finished(return_code, event_name)
```

Comment:
Changing the order of these parameters needs to be reflected in eventual yields() returned values that await this signal. There are a few in esc_room.gd (L167), esc_prompt_popup.gd (L36), escoria.gd (L342)

Reply:
Actually, it doesn't. It was wrong all the time. We send (return_code, event_name), but the signal definition was turned around. :)

Reply:
Oh, shoot 😮 
All good then ;) Thanks!